### ISSUE TITLE
Callback processRowPostCreate gets called before row added to DOM

### ISSUE THREAD

**Issue by h3nn3** [ag-grid/ag-grid#1633]
**I'm submitting a feature request**
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/ceolter/ag-grid/blob/master/CONTRIBUTING.md#question
```
I need a callback after a row or all rows have been added to the DOM. However the callback processRowPostCreate gets called to early. I want to resize the row height based on its highest column content (richtext). At the moment i call my resize function in celldidupdate (react) after editing but i need a hook for calling my resize function initally after a row has been added to the dom (virtualization). Is there any possibility to achieve this?

Thx :)

**Comment by makinggoodsoftware** [comment]
Hi, 

Thank you for raising this issue for us, at this moment we are reviewing how we process Feature Requests/questions that come through from GitHub.

As I am sure you are aware, we offer two products:

ag-Grid - this is entirely free to use in line with our MIT license
ag-Grid Enterprise - our commercial version which funds the development of both versions.

We currently have a very full pipeline of work which can be viewed here: https://www.ag-grid.com/ag-grid-pipeline/

This will be our priority for the coming months and as a result, we don’t have capacity to take feature requests/questions from GitHub, but you might find that yours has already been raised in our pipeline.

We will do a periodic review of GitHub issues for any bugs raised and these can also be tracked on our pipeline page.

Thanks for your understanding.
ag-Grid Team.

**Comment by brendonco** [comment]
Hi, which AG-GRID/Enterprise version are you facing this issue? is it 18.0.1?

@makinggoodsoftware which version of ag-grid that have processRowPostCreate fixed?  I'm currently using 18.0.1 AG-GRID/Enterprise version.  I saw in v19.1.4 which have similar event called firstDataRendered.  I want to listen to an event when AG-GRID is loaded in DOM and have data loaded before I do any data manipulation in AG-GRID. How do I do that in v18.01?  Please advise.
